adserver/templates/adserver/reports/includes/publisher-report-table.html

```diff
@@ -8,13 +8,17 @@
       <th class="text-right"><strong>{% trans 'Views' %}</strong></th>
       <th class="text-right"><strong>{% trans 'Clicks' %}</strong></th>
       <th class="text-right"><strong>{% blocktrans %}<abbr title="Click through rate">CTR</abbr>{% endblocktrans %}</strong></th>
+      <th class="text-right"><strong>{% blocktrans %}<abbr title="% of traffic that viewed the ad">View Rate</abbr>{% endblocktrans %}</strong></th>
+      {% if publisher.saas %}
+      <th class="text-right"><strong>{% blocktrans %}<abbr title="Billable API requests">API Requests</abbr>{% endblocktrans %}</strong></th>
```

Comment:
This explanation feels like it could be slightly improved but hard for me to come up with something concretely better.

Reply:
Yea... I think we need a reference doc that explains all these in more depth. Trying to put them in a tooltip is a losing battle.